grant-permissions.bat

```diff
@@ -0,0 +1,49 @@
+@echo off
+adb version
+echo MeldCX note: ADB Version must be 1.0.33 or above
+
+adb root
+timeout /t 20 /NOBREAK
```

Comment:
After adb root, do we need 20 seconds to sleep? I feel 1 or 2 seconds is enough

Reply:
Cool, I will reduce to 5s then, I notice it took more than 3s on my side sometimes.